packages/webapp/src/lib/components/WithdrawModal.svelte

```diff
@@ -46,10 +47,13 @@
 
 	function handleClose() {
 		open = false;
-		amount = 0n;
+		amount = '0';
 	}
 
-	$: validation = validateAmount(amount, BigInt(vault.balance));
+	$: validation = validateAmount(
+		parseUnits(amount, Number(vault.token.decimals)),
```

Comment:
why can't we do this with floats rather than having to convert everything back to bigint?